requirements.in

```diff
@@ -37,7 +37,7 @@ drf-spectacular                         # DRF API documentation
 feedparser                              # RSS newsfeed parser
 gunicorn                                # Gunicorn web server
 pdf2image                               # PDF to image conversion
-pillow==9.5.0                           # Image manipulation # FIXED 2023-07-04 as we require PIL.Image.ANTIALIAS
+pillow                                  # Image manipulation
```

Comment:
@SchrodingersGat Unfortunately this line change breaks the inventree-brother-plugin due to brother_ql needing a fix seen in https://github.com/pklaus/brother_ql/pull/143. Any suggestions on how to get this back to working?

Reply:
@atanisoft we are using a fork of brother_ql  - https://github.com/matmair/brother_ql-inventree so we can just update that

Reply:
That would be great if that could be updated, can a new version be pushed for both to pick up that fix?

Reply:
New version is released, 1.1 includes the fix and seems to work

Reply:
Thanks, after shutting down and starting up the docker containers it seems to have fixed it with the newer version.

Reply:
@atanisoft good to hear! And @matmair thanks for the quick fix